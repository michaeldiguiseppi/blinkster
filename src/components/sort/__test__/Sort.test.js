import React from 'react';
import { Sort } from '../Sort';
import renderer from 'react-test-renderer';
import { shallow, configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('<Sort /> component', () => {
    const defaultProps = {}

    it('should render without breaking', () => {
        const sortComponent = mount(<Sort {...defaultProps} />);
        expect(sortComponent.exists()).toBe(true);
        sortComponent.unmount();
    });

    it('should match the snapshot', () => {
        const tree = renderer
            .create(<Sort {...defaultProps} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});