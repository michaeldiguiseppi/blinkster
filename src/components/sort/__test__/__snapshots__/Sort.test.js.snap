// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sort /> component should match the snapshot 1`] = `<div />`;
